Create a directory to keep all the overview notebooks.

alarshi opened the pull request:
This PR adds an overview directory which contains the first notebook (overview) of each of the modules.

Commit:
Create a directory to keep all the overview notebooks.

ljhwang commented:
Great! Thanks for the additional sections and summary.  

Will the table of contents have hyperlinks to the section?

Please also correct multiple)(2) typos in the "tranfer" == "transfer" in the Objectives.

Commit:
Fix typos.

alarshi commented:
@ljhwang : Thanks for the review!
I fixed the typos in the latest commit.

Unfortunately, I don't think it is possible to add hyperlinks to section titles across different notebooks :-/ But, I can add the notebook links, what do you think?

ljhwang commented:
I was just thinking about links to the notebooks note the sections.  

There are ways to include section links but I do not know if it works across notebooks:
https://sebastianraschka.com/Articles/2014_ipython_internal_links.html

naliboff commented on bin/overview-modules/0_overview-heat-flow-module.ipynb:
```diff
@@ -0,0 +1,210 @@
+{
```
<blockquote>Upon completion of this module you would know how to use python to:</blockquote><p>Replace "would" with "will"?</p><p> </p><blockquote>An introduction to the conductive heat transfer mechanism within the Earth.</blockquote><p>Suggestion: "An introduction to conductive heat transfer within the Earth."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

naliboff replied:
<blockquote>An introduction to <strong>the </strong>conductive heat transfer mechanism within the Earth. Add bolded word (<strong>the</strong>).</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/6/discussion/#comment-1806896991'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

naliboff commented:
@alarshi - Looks good, just a few minor comments.

Commit:
Address changes from the PR.

alarshi commented:
@naliboff Thank you for the suggestions! I addressed them in the latest commit.
@ljhwang : I don't think we can add section references across notebooks :( . I just added the links to the notebooks, hopefully that is enough for our purposes.

naliboff commented:
@alarshi - Just one more minor comment, and then this will be ready to merge.

Commit:
Fix grammar.

alarshi commented:
@naliboff : Thanks, just fixed it! :)
